operator/src/main/resources/base-keycloak-deployment.yaml

```diff
@@ -29,14 +29,25 @@ spec:
         livenessProbe:
           exec:
             command:
-              - curl --head --fail --silent http://127.0.0.1:8080/health/live
+              - curl
+              - --head
+              - --fail
+              - --silent
+              - http://localhost:8080/health/live
           periodSeconds: 1
+          failureThreshold: 10
+          initialDelaySeconds: 30
         readinessProbe:
           exec:
             command:
-              - curl --head --fail --silent http://127.0.0.1:8080/health/ready
+              - curl
+              - --head
+              - --fail
+              - --silent
+              - http://localhost:8080/health/ready
           periodSeconds: 1
-          failureThreshold: 180
+          failureThreshold: 10
+          initialDelaySeconds: 30
```

Comment:
We will possibly start sooner than in 30 secs.

Reply:
not early than 20 seconds on my machine with augmentation ...

Reply:
👍  Please resolve this issue.